packages/valory/skills/registration_abci/tests/test_behaviours.py

```diff
@@ -460,6 +461,7 @@ def test_request_update(
     ) -> None:
         """Test Tendermint config update"""
 
+        self.state.updated_genesis_data = {}
```

Comment:
If you look at the tests 

https://github.com/valory-xyz/open-autonomy/blob/70f4889c39789ace0f0da52bea4da6634e773358/packages/valory/skills/registration_abci/tests/test_behaviours.py#L457-L505

There are basically 4 tests, `test_request_update[True, False]` and `test_request_restart[True, False]`. The `[True, False]` are parametrised as `valid_response` variable. Now every time the test ran only one of these 4 tests passed and other 3 failed depending on which test ran first. The first test passed and the rest failed because of the following block of code

https://github.com/valory-xyz/open-autonomy/blob/main/packages/valory/skills/registration_abci/behaviours.py#L385-L390

Now, since `state` variable is shared among all of the tests when first test ran it populated the `updated_genesis_data`. And since the `updated_genesis_data` is populated for the rest of the tests they never reached to the point of http request and caused the test to fail

Reply:
Shouldn’t this reset happen in “setup” of the test?

Reply:
I was not sure if that would effect on other tests, I'll give it a check